doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -143,7 +145,19 @@ \subsection{AvailabilityManager:NightCycle}\label{availabilitymanagernightcycle}
 
 \textbf{\emph{Field: Control Type}}
 
-The possible inputs are \emph{StayOff}, \emph{CycleOnAny}, \emph{CycleOnControlZone}, \emph{CycleOnAnyZoneFansOnly}. \emph{StayOff} means the availability manager will have no effect air handler on/off will be determined by the fan schedule. \emph{CycleOnAny} means that if any zone served by a system whose AvailabilityManagerAssignmentList references this system availability manager has an air temperature outside the cooling or heating set points the central fan will turn on even though the fan schedule indicates the fan is off. \emph{CycleOnControlZone} means the same thing except the availability manager looks at the temperature in only one zone. \emph{CycleOnAnyZoneFansOnly} is the same as \emph{CycleOnAny} except that only the zone fans are cycled on and the central air handler fan is left off. The default is \emph{StayOff}.
+The possible inputs are \emph{StayOff}, \emph{CycleOnAny}, \emph{CycleOnControlZone}, \emph{CycleOnAnyZoneFansOnly}, \emph{CycleOnAnyCoolingOrHeatingZone}, \emph{CycleOnAnyCoolingZone}, \emph{CycleOnAnyHeatingZone}, or \emph{CycleOnAnyHeatingZoneFansOnly}. The default is \emph{StayOff}.
+
+\begin{itemize}
```

Comment:
Just a note, in addition to the `itemize` environment, there is also the [`description`](https://en.wikibooks.org/wiki/LaTeX/List_Structures#List_structures) environment that can do this sort of "keyword followed by description" structure.  Definitely no need to change anything, just noting it for future doc updates.